tests/sflow/test_sflow.py

```diff
@@ -109,6 +109,28 @@ def config_dut_ports(duthost, ports, vlan):
     config_reload(duthost, config_source='config_db', wait=120)
     time.sleep(5)
 
+# ----------------------------------------------------------------------------------$
+
+
+def get_default_agent(duthost):
+    # get_default_agent function is used to get the agentip selected by the hsflowd dameon.
+    # hsflowd.auto is the auto generated file by hsflowd daemon
+    hsflowd_autogen_file = '/etc/hsflowd.auto'
+    result = duthost.shell(
+                        r"docker exec -i sflow sh -c 'test -f {} && echo exist'".format(hsflowd_autogen_file),
+                        module_ignore_errors=True)['stdout'].strip()
+    if result != 'exist':
+        pytest.fail("{} file doesnot exist.".format(hsflowd_autogen_file))
```

Comment:
Can you help to change the typo, when i cherry-pick to our local, it failed in our spell checker

doesnot ---> does not